examples/using_mindsdb_inference_with_text2sql_prompt.py

```diff
@@ -0,0 +1,95 @@
+from openai import OpenAI
+from mindsdb_sdk.utils.openai import extract_sql_query, query_database, chat_completion_request, \
```

Comment:
This is a purely prompt based approach and doesn't use tools, it would allow us to work with any supported LLM in litellm without the need for tools callback